pycernan/avro/metrics.py

```diff
@@ -0,0 +1,24 @@
+from prometheus_client.metrics import Counter, Histogram
+
+PREFIX = "pycernan"
+
+
+def p(n):
```

Comment:
I am unsure what `p` means. Can we set something more verbose ?

Reply:
Just prefix, it's only used here and for naming.